conf/apache/rais.conf

```diff
@@ -1,9 +1,7 @@
 # RAIS IIIF URLs
 AllowEncodedSlashes NoDecode
 
-ProxyPassMatch ^/images/iiif/(.*(?:jpg|info\.json))$ http://localhost:12415/rais/$1 nocanon
-
-ProxyPassMatch ^/images/resize/([^/]*)/full/([0-6][0-9][0-9],.*jpg)$ http://localhost:12415/rais/$1/full/$2 nocanon
```

Comment:
If we drop this, we should drop the apache-based file caching (or else show an example of how it could still work, though I think that's probably out of scope)

Reply:
Way ahead of you on this: https://github.com/open-oni/open-oni/issues/464

Reply:
Apache-based caching removed